app/models/configuration_architecture.rb

```diff
@@ -1,2 +1,5 @@
 class ConfigurationArchitecture < ConfigurationTag
+  def self.base_model
+    ConfigurationArchitecture
+  end
```

Comment:
NOTE the tag_hash uses the class name [[here]](https://github.com/ManageIQ/manageiq/pull/21597/files#diff-f1ec1aecc738887fbcdf4fd7572208afd9259fb2feae88f602c16efe2f9af208L71) which fails the hash lookup when the class is namespaced.